qcodes/instrument_drivers/QDevil/QDevil_QDAC.py

```diff
@@ -49,9 +52,9 @@ class Waveform:
     all_waveforms = [sine, square, triangle, staircase]
 
 
-class Generator():
+class Generator:
     #  Class used in the internal book keeping of generators
-    def __init__(self, generator_number):
+    def __init__(self, generator_number: int):
         self.fg = generator_number
         self. t_end = 9.9e9
```

Comment:
this line seems to have this extra space - did't know that this happens to be valid python :)
```suggestion
        self.t_end = 9.9e9
```